meetings/2022-02-24.md

```diff
@@ -35,6 +35,9 @@
 
 ## Topics
 
+This meeting was not scribed. The topics below reflects only the agenda but not the discussion.
```

Comment:
```suggestion
This meeting was not scribed. The topics below reflect only the agenda, not the discussion.
```

Reply:
You could argue in fact that there is a  value of the minutes which is proportional to the number of people who did NOT attend!